qiskit_nature/operators/second_quantization/second_quantized_op.py

```diff
@@ -31,31 +30,31 @@ class SecondQuantizedOp(StarAlgebraMixin):
     def __init__(self, operator_list: List[ParticleOp]):
         """
         """
-        # TOOD: validation of operator_list
+        # TODO: validation of operator_list
 
-        self._fermion = None
-        self._boson = None
-        self._spin: Dict[int, SpinOp] = {}
+        self.fermion = None
```

Comment:
I guess you have made these public to allow them to be part of public API and settable/gettable directly and did not want to add getter/setters as is done elsewhere in qiskit. Documentation is a concern here since how Sphinx is setup for Qiskit the setters/getters are documented but instance vars are not even if they have comments on them. I would most likely vote for maintaining consistency and having these are properties to expose what were the private marked vars. Things can be changed throughout Qiskit later once Sphinx etc are done to cater for public instance vars. It should be a simple code code with no impact to end users code to go from properties to public vars.

Reply:
Generally, we will need to access these specific fields in various arguments (as far as I understand). I can make them properties for the sake of being compliant.

Reply:
I have to agree with Steve here. Please stick to using properties for now and keep the instance variables private.